src/models/category.js

```
@@ -0,0 +1,38 @@
+  DOCUMENT_ALREADY_EXISTS,
+  FIELD_CANNOT_BE_SHORTER,
+  FIELD_CANNOT_BE_LONGER
+} = require('~/consts/errors')
+const { CATEGORY } = require('~/consts/models')
+const {
+  regex: { HEX_COLOR_CODE_PATTERN }
+} = require('~/consts/validation')
+
+const categorySchema = new Schema({
+  name: {
+    type: String,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)